model_analyzer/config/config.py

```diff
@@ -470,6 +470,74 @@ def _setup_logger(self):
                             "[%(filename)s:%(lineno)d] %(message)s",
                             datefmt="%Y-%m-%d %H:%M:%S")
 
+    def _autofill_values(self):
+        """
+        Fill in the implied or default
+        config values.
+        """
+
+        new_model_names = {}
+        for model in self.model_names:
+            new_model_names[model.model_name()] = {}
+            if not model.objectives():
+                if 'objectives' not in self._fields:
```

Comment:
`objectives` is an optional field. We don't have to specify it. I don't think we should raise an exception when it is not defined.